### ISSUE TITLE
mAP is very low

### ISSUE THREAD

**Issue by lishiyu93** [ciderpark/Libtorch_YOLOv3_train_demo#1]
Hi,recently, I used your code to train my own data set (train images:192,evaluate images:21,class number:1). I trained 1000 epochs. The maximum mAP value does not exceed 0.5, and the fluctuation is very large.I used the same data set and training parameters to train on the Python version of Yolov3( https://github.com/ultralytics/yolov3 ). About 150 epochs were trained, and the mAP reached more than 0.8.Could you please give me some good Suggestions,best wishes for you !

**Comment by ciderpark** [comment]
Hi, I got this issue before, I changed the learning rate to 1e-5, and train without weight_decay. Then, it worked extremely well. I got the mAP reached 0.93 on two classes object detection task.




Best wishes!





At 2020-11-09 11:04:45, "lishiyu93" <notifications@github.com> wrote:

Hi,recently, I used your code to train my own data set (train images:192,evaluate images:21,class number:1). I trained 1000 epochs. The maximum mAP value does not exceed 0.5, and the fluctuation is very large.I used the same data set and training parameters to train on the Python version of Yolov3( https://github.com/ultralytics/yolov3 ). About 150 epochs were trained, and the mAP reached more than 0.8.Could you please give me some good Suggestions,best wishes for you !

**Comment by ciderpark** [comment]
I trained 100 epoches.


Best wishes!

















At 2020-11-09 11:04:45, "lishiyu93" <notifications@github.com> wrote:

Hi,recently, I used your code to train my own data set (train images:192,evaluate images:21,class number:1). I trained 1000 epochs. The maximum mAP value does not exceed 0.5, and the fluctuation is very large.I used the same data set and training parameters to train on the Python version of Yolov3( https://github.com/ultralytics/yolov3 ). About 150 epochs were trained, and the mAP reached more than 0.8.Could you please give me some good Suggestions,best wishes for you !
